resources/views/article/show.blade.php

```diff
@@ -0,0 +1,37 @@
+<x-layout>
+    <x-section>
+        <h4>
+            <a href="{{ url()->previous() }}">
+                &lt; Go Back
+            </a>
+        </h4>
+        <article class="space-y-2 my-4">
+            <figure>
+                <img src="{{ str_starts_with($article->image, 'https')
```

Comment:
Maybe add this logic to [getter](https://laravel.com/docs/8.x/eloquent-mutators#defining-an-accessor) instead of doing in blade file

Reply:
Adding a getter is perfect for this, i didn't thought of that.
I was even thinking of making it a blade component at some point, but a getter is way better.